doc/src/manual/complex-and-rational-numbers.md

```diff
@@ -135,12 +138,11 @@ Note that mathematical functions typically return real values when applied to re
 complex values when applied to complex numbers. For example, [`sqrt()`](@ref) behaves differently
 when applied to `-1` versus `-1 + 0im` even though `-1 == -1 + 0im`:
 
-```julia
+```jldoctest
 julia> sqrt(-1)
 ERROR: DomainError:
 sqrt will only return a complex result if called with a complex argument. Try sqrt(complex(x)).
- in sqrt(::Int64) at ./math.jl:278
```

Comment:
Same comment here as https://github.com/JuliaLang/julia/pull/20202#discussion_r97447735

The point of the prompt-output style of these runnable code examples is to be representative of what you see when you run these yourself in the REPL.